config/zones/CA-ON.yaml

```
@@ -210,8 +221,6 @@ parsers:
   price: CA_ON.fetch_price
   production: CA_ON.fetch_production
 sources:
-  ? "Mallia, E., Lewis, G. \"Life cycle greenhouse gas emissions of electricity generation\
-    \ in the province of Ontario, Canada.\" Int J Life Cycle Assess 18, 377\u2013\
-    391 (2013)"
+  ? "Mallia, E., and G. Lewis. “Life Cycle Greenhouse Gas Emissions of Electricity Generation in the Province of Ontario, Canada.” The International Journal of Life Cycle Assessment, vol. 18, no. 2, 2012, pp. 377–391, https://doi.org/10.1007/s11367-012-0501-0."
```

```suggestion
  Mallia, E., and G. Lewis. "Life Cycle Greenhouse Gas Emissions of Electricity Generation in the Province of Ontario, Canada." The International Journal of Life Cycle Assessment, vol. 18, no. 2, 2012, pp. 377–391:
    link: https://doi.org/10.1007/s11367-012-0501-0
```
Don't forget to remove the duplicate link under this line.